app/test_main.py

```diff
@@ -1,3 +1,22 @@
+import pytest
+
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_age, dog_age, expected",
+    [
+        (0, 0, [0, 0]),
+        (100, 100, [21, 17]),
+        (14, 14, [0, 0]),
+        (23, 23, [1, 1]),
+        (24, 24, [2, 2]),
+        (27, 27, [2, 2]),
+        (28, 28, [3, 2]),
+
+    ]
+)
+def test_cat_dog_years_to_human(cat_age: int,
+                                dog_age: int, expected: list[int]) -> None:
```

Comment:
Here can be better functioning styling, like this:
```
def test_cat_dog_years_to_human(
    cat_age: int, 
    dog_age: int, 
    expected: list[int]
) -> None:
```

Use a one liner for arguments if it is possible, if not -> one argument per line